Makefile

```diff
@@ -60,7 +60,7 @@ test:
 	@make test-server
 
 test-server:
-	@docker exec -t sc-flask pytest server/
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **🚨 issue (security):** Adding `-s` flag to pytest may expose sensitive information in logs.

The `-s` flag disables all capturing. Ensure that no sensitive data is printed to the console as it could be exposed in CI/CD logs.